src/main/java/org/jalasoft/moi/model/core/IHandler.java

```diff
@@ -22,4 +27,15 @@
      * @return a String of result from CommandBuilder and Executer handling
      */
     String execute(Params params);
+
+    /**
+     * Returns a Params object.
+     * A JSON object gets deconstructed and its data used to make a
+     * Params object.
+     * @param jsonRequest A string ready to be decomposed into variables.
+     * @return A Params object.
+     * @throws IOException
+     * @throws ParseException
+     */
+    Params codeParams(String jsonRequest) throws IOException, ParseException;
```

Comment:
I believe that it should have a more significant name
I suggest: `convertToParams(...)`